### DISCUSSION TITLE
Eclipse import project error

### DISCUSSION THREAD

**Discussion by yunkuotianxian** [dbeaver/dbeaver#16551] [DBeaver Development]
![image](https://user-images.githubusercontent.com/18697373/169529455-df71cf37-d4ae-4c50-b4e4-88ff4353ce7e.png)

#### System information: 
- Operating system (distribution) and version
- DBeaver version
- Java version 
- Additional extensions

#### Connection specification:
- Database name and version
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)?

**Comment by LonwoLonwo** [comment]
Do you use the last Eclipse version?

**Comment by yunkuotianxian** [comment]
Yes, the latest version, just downloaded

Eclipse IDE for RCP and RAP Developers - 2022-03







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dbeaver/dbeaver"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月20日(星期五) 晚上8:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dbeaver/dbeaver] Eclipse import project error (Discussion #16551)





 
Do you use the last Eclipse version?

**Reply by E1izabeth** [reply]
Please, accept Resolve Later action for all problems and then try to use Update Maven Project with Force Update of Snapshots/Releases
![image](https://user-images.githubusercontent.com/28875055/170129135-5b28b17d-bc0d-4342-bbe2-3cf9d4ffdb85.png)
![image](https://user-images.githubusercontent.com/28875055/170129246-5aeb986c-7c7b-4ca0-95cc-524e7a5fd683.png)

Does it help?

**Comment by yunkuotianxian** [comment]
I did what you said, but I still couldn't. It's the same as the previous error report.


If the eclipse IDE is replaced by eclipse-rcp-2021-12, the same error is reported. I tried to clone other branches, but the same error is reported.


Is there a lack of jar package in Maven repository? Someone should encounter the same problem as me. How do you do it?







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dbeaver/dbeaver"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月25日(星期三) 凌晨4:49
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dbeaver/dbeaver] Eclipse import project error (Discussion #16551)





 
Please, accept Resolve Later action for all problems and then try to use Update Maven Project with Force Update of Snapshots/Releases
 
 
 
Does it help?

**Comment by E1izabeth** [comment]
The reason for this problem is tycho version update https://github.com/dbeaver/dbeaver/commit/b4f028dc06fd342c0c394010f9f407d34ae11c75

**Comment by yunkuotianxian** [comment]
It still doesn't work to restore to the previous version. What are your versions of eclipse and Maven?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dbeaver/dbeaver"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月25日(星期三) 下午3:26
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dbeaver/dbeaver] Eclipse import project error (Discussion #16551)





 
The reason for this problem is tycho version update b4f028d

**Comment by ShadelessFox** [comment]
Try changing `Resolve Later` to `Do Not Execute (add to workspace)` in the `Action` column for all failed plugins.

**Comment by yunkuotianxian** [comment]
I did as you said. The project reported an error. It seems that the plug-ins that cannot be loaded must be loaded







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "dbeaver/dbeaver"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月25日(星期三) 晚上7:44
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [dbeaver/dbeaver] Eclipse import project error (Discussion #16551)





 
Try changing Resolve Later to Do Not Execute (add to workspace) in the Action column for all failed plugins.

**Comment by yunkuotianxian** [comment]
I succeeded!

Put dbeaver / POM The in XML is modified as follows:

<configuration>
  <useProjectSettings>false</useProjectSettings>
  <source>11</source>
  <target>11</target>
  <compilerVersion>11</compilerVersion>
  ................
</configuration>

**Comment by miltonchirinos44** [comment]
Tuve el mismo problema,pero al parecer ya lo solucionaste @yunkuotianxian
